core/src/main/java/org/apache/accumulo/core/util/threads/ThreadPools.java

```diff
@@ -474,6 +474,8 @@ public ThreadPoolExecutor createThreadPool(int coreThreads, int maxThreads, long
   public ThreadPoolExecutor createThreadPool(int coreThreads, int maxThreads, long timeOut,
       TimeUnit units, final String name, BlockingQueue<Runnable> queue, OptionalInt priority,
       boolean emitThreadPoolMetrics) {
+    LOG.debug("Creating ThreadPoolExecutor for {} with {} core threads and {} max threads", name,
+        coreThreads, maxThreads);
```

Comment:
```suggestion
    LOG.debug(
        "Creating ThreadPoolExecutor for {} with {} core threads and {} max threads {} {} timeout",
        name, coreThreads, maxThreads, timeOut, units);
```

During testing of scan servers under high load I was seeing really erratic behavior. I noticed that the thread pool that serviced RPC request was cycling through lots of threads (like 1,000's of thread being created and destroyed in the pool per minute).  Turns out the thread pool timeout defaults to 10s.  I changed it to zero in my config for no thread pool timeout and the performance problems and erratic behavior went away.  I don't fully understand the problem at this point, but adding the timeout to this debug is useful.

Reply:
This is where I configured the timeout to zero for my scan servers to avoid this problem.  

https://github.com/keith-turner/accumulo-testing/blob/322b32c62113b974b9dd18404d0e57902dc13b8e/sstest/accumulo-scanservers.yaml#L19

Reply:
Opened #2745

Reply:
It looks like SSERV_MINTHREADS_TIMEOUT defaults to 10s, but TSERV_MINTHREADS_TIMEOUT defaults to 0s.